src/shared/models/game/Placement.es6

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+//
+// Player
+// Created by dpekar on 10/8/15.
+//
+
+const BaseModel = require('../BaseModel')
+
+class Placement extends BaseModel {
+    constructor(tile, coordinates, turn) {
```

Comment:
It's the number of turns into the game this tile was placed on.  Might be useful for things like Restaurants, SkyScraper. 

Changed to turnNum. To be more clear.